bodhi/server/templates/update.html

```diff
@@ -689,6 +689,15 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              <div class="p-b-1">
+                <div>
+                  <strong>CI Status</strong>
```

Comment:
Oh I see, you're totally right and sorry for missing what you were saying, we were talking about two different places.